services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/command/CommandContext.java

```diff
@@ -42,6 +42,10 @@ public long getCtime() {
         return ctime;
     }
 
+    public void setCorrelationId(String correlationId) {
```

Comment:
This is "correlationId" recovery mechanism. I.e. first command send some packet via FL in "network", this packet contain "correlation-id" of original command. It will be good to track following `operation` with same "correlation-id". But it can't be set by executor, because it must be extracted from packet first... but at this moment we already have some context with some "correlation-id".

This method is first step for `smooth` replacement of correlation-id, with correct tracking in logs of this replacement.